tests/src/main/java/com/hazelcast/stabilizer/tests/map/MapHeapHogTest.java

```diff
@@ -1,116 +1,149 @@
 package com.hazelcast.stabilizer.tests.map;
 
-
 import com.hazelcast.core.HazelcastInstance;
 import com.hazelcast.core.IMap;
 import com.hazelcast.logging.ILogger;
 import com.hazelcast.logging.Logger;
 import com.hazelcast.stabilizer.test.TestContext;
-import com.hazelcast.stabilizer.test.annotations.Run;
+import com.hazelcast.stabilizer.test.annotations.RunWithWorker;
 import com.hazelcast.stabilizer.test.annotations.Setup;
 import com.hazelcast.stabilizer.test.annotations.Verify;
 import com.hazelcast.stabilizer.test.annotations.Warmup;
-import com.hazelcast.stabilizer.test.utils.ThreadSpawner;
+import com.hazelcast.stabilizer.worker.tasks.AbstractMonotonicWorkerTask;
+import com.hazelcast.stabilizer.worker.tasks.AbstractWorkerTask;
 
 import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicInteger;
 
 import static com.hazelcast.stabilizer.test.utils.TestUtils.humanReadableByteCount;
 import static com.hazelcast.stabilizer.tests.helpers.HazelcastTestUtils.isMemberNode;
 import static com.hazelcast.stabilizer.tests.helpers.HazelcastTestUtils.nextKeyOwnedBy;
+import static com.hazelcast.stabilizer.utils.CommonUtils.sleepSeconds;
+import static java.lang.String.format;
 
 public class MapHeapHogTest {
 
-    private final static ILogger log = Logger.getLogger(MapHeapHogTest.class);
+    private static final int MIN_CLUSTER_SIZE = 3;
+    private static final long APPROX_ENTRY_BYTES_SIZE = 238;
+    private static final ILogger LOGGER = Logger.getLogger(MapHeapHogTest.class);
 
     // properties
-    public String basename = this.getClass().getName();
+    public String basename = this.getClass().getSimpleName();
     public int threadCount = 3;
     public int ttlHours = 24;
     public double approxHeapUsageFactor = 0.9;
+    public int logFrequency = 500000;
+    /**
+     * Configures the worker threads to use the same keys, which slows down the fill rate while keeping the load high.
+     */
+    public boolean useSameKeysOnAllWorker = true;
```

Comment:
`threadCount` was set to 3, so we had 3 local worker threads per member, but all were using the same local keys (start with 0, increase by 1):
```
for (i < maxKeys) {
  key = 0;
  map.put(key, key);
  findNextLocalKey(key);
  key += 1;}
```

So by increasing the `threadCount` we just spawned multiple workers which did exactly the same thing. Now we can configure (or change this as default) that each worker should use a different local key (start with 0 -> threadCount, increase by threadCount):
```
for (i < maxKeys) {
  key = atomicInt.getAndIncrement();
  map.put(key, key);
  findNextLocalKey(key, threadCount);
  key += threadCount }
```

This will speed up the inserts. If you did not intend to just create more load on the cluster with multiple workers, we can make this the default behavior and drop that property for that.